integration/smartcontract/pk_test.go

```diff
@@ -15,6 +15,8 @@ func TestSign(t *testing.T) {
 	}
 	pubKeyA := privateKeyA.PublicKey
 	addrA := crypto.PubkeyToAddress(pubKeyA)
+	t.Logf("%x\n", privateKeyA.D.Bytes())
```

Comment:
@otherview Did you leave this in on purposes?

Reply:
I did. I want it to log the keys.

I've seen issues where keys don't work, want it to have keys logged in case that test fails.

Reply:
Is it worth saying what it is in the log msg then? Future users would appreciate this I think.

Reply:
Yes, you're right 👍